example_app_generator/generate_app.rb

```diff
@@ -33,7 +33,7 @@
   end
 
   if Rails::VERSION::STRING >= "5.1.0"
-    gsub_file "Gemfile", /.*selenium-webdriver.*/, "gem 'selenium-webdriver', '<= 3.14'"
+    gsub_file "Gemfile", /.*chromedriver-helper.*/, "gem 'webdrivers'"
```

Comment:
I think we should still keep specifying a version.

Reply:
Keep specifying a version of what? This just reverts the selenium-webdriver specification to whatever it was before https://github.com/rspec/rspec-rails/commit/e10484e3ea3352afeac5edfcc62ab456f357d53c which was done because of the deprecation notices emitted by the latest selenium-webdriver.  Beyond that `webdrivers` specifies it's compatibility with `selenium-webdriver` drivers and is added just like in Rails now (which is without any specific version specified).

Reply:
I am ok with this then.